### PR TITLE
🐛 Fix underflows in examples

### PR DESCRIPTION
**Pull request by 0xBeans**
invariant: block.timestamp >= startTime

### CODE REVIEW

File: src/examples/LinearNFT.sol [hunk 1]
```diff
@@ -43,8 +43,8 @@ contract LinearNFT is ERC721, LinearVRGDA {
     //////////////////////////////////////////////////////////////*/
 
     function mint() external payable returns (uint256 mintedId) {
-        // Note: By using toDaysWadUnsafe(startTime - block.timestamp) we are establishing that 1 "unit of time" is 1 day.
-        uint256 price = getVRGDAPrice(toDaysWadUnsafe(startTime - block.timestamp), mintedId = totalSold++);
+        // Note: By using toDaysWadUnsafe(block.timestamp - startTime) we are establishing that 1 "unit of time" is 1 day.
+        uint256 price = getVRGDAPrice(toDaysWadUnsafe(block.timestamp - startTime), mintedId = totalSold++);
```
Comment (transmissions11) [inline src/examples/LinearNFT.sol:47]: should uncheck this line too!

Comment (0xBeans) [reply src/examples/LinearNFT.sol]: Nice, done. Lmk if styling needs changing

Comment (transmissions11) [reply src/examples/LinearNFT.sol]: yeah would prefer to include everything in one unchecked block now that we have no math statements we need to be checked

Comment (0xBeans) [reply src/examples/LinearNFT.sol]: coooool

**Pull request merged**